src/components/PuzzleGrid/index.jsx

```diff
@@ -13,7 +13,7 @@ const NUM_OF_TILES = 6;
 // a user can click on one tile to select and then click on another tile to swap their places
 // the user will swap the places of each tile to put it in its correct position
 // when all tiles are placed in their correct position the full image is displayed
-const PuzzleGrid = () => {
+const PuzzleGrid = ({ puzzledImage }) => {
```

Comment:
puzzled is not really a verb in the sense of "making something a puzzle" so puzzleImage would be more clear. a puzzled image would be a confused image in english ;)

Reply:
Oh my bad :sweat_smile:. fixed PTAL